jqqb/rule.py

```diff
@@ -15,19 +15,19 @@ def __init__(self, rule_dict):
         self.value = rule_dict["value"]
 
     def evaluate(self, obj):
-        result = self.get_operator()(self.get_input(obj), self.get_value())
+        result = self.get_operator()(self.get_input(obj), self.get_value_customFields(obj))
         return result
 
     def inputs(self, obj):
         inputs = self.get_input(obj)
         return inputs
 
-    def values(self):
-        values = self.get_value()
+    def values(self, obj):
```

Comment:
It could be interesting to add a default value to `obj`, that will avoid to break all interface contracts.